src/extensionProvider.ts

```diff
@@ -1,7 +1,9 @@
 import { ISignableMessage, ITransaction } from "./interface";
 import { Address, Signature } from "./primitives";
 import { Operation } from "./operation";
-import { ErrCannotSignSingleTransaction } from "./errors";
+import { ErrAccountNotConnected, ErrCannotSignSingleTransaction } from "./errors";
+
+const disconnectedAccount = { address: "" };
```

Comment:
It took me a while to catch the trick with `disconnectedAccount`. Wouldn't it be more straight forward to have a method `disconnect()` or `disconnectAccount()`?